config/runtime/error_handler.go

```diff
@@ -12,9 +12,33 @@ import (
 	"github.com/avenga/couper/handler"
 )
 
-var wildcardMap = map[string][]string{
-	"api":      {"backend_openapi_validation", "backend_timeout", "beta_insufficient_permissions"},
-	"endpoint": {"backend_openapi_validation", "backend_timeout", "beta_insufficient_permissions", "sequence", "unexpected_status"},
+type wildcardValues map[string][]string
+
+var wildcardMap = map[string]wildcardValues{
```

Comment:
This is not used exclusively for wildcards any more. It now maps another super-kind: "backend". `superkindValues` and `superkindMap`? Also see other variables and terms in lines 60ff.

The flattened hierarchy may be more performant. However, there is some repetition now, which may become confusing and error-prone, e.g. if "backend_openapi_validation" becomes a super-kind for, say, "backend_openapi_request_validation" and "backend_openapi_response_validation" in the future.

If we want to keep this flattened hierarchy, it should be generated (like the stuff in errors/types_generated.go).

What about the tests in my previous comment?